wallet-new/test/unit/DeltaCompressionSpecs.hs

```diff
@@ -0,0 +1,141 @@
+{-# LANGUAGE RankNTypes #-}
```

Comment:
Are you sure you need to add all these language extensions? In the case of `wallet-new:cardano-unit-tests` if I'm not mistaken we already add this extension in the default ones:

```
  default-extensions: BangPatterns
                      ConstraintKinds
                      DeriveGeneric
                      FlexibleContexts
                      FlexibleInstances
                      GADTs
                      InstanceSigs
                      LambdaCase
                      MultiParamTypeClasses
                      MultiWayIf
                      NoImplicitPrelude
                      OverloadedStrings
                      RankNTypes
```

Just wondering, is this in order to your editor to type check your code? (Asking as in the past I had to do similar tricks for stylish-haskell or `ghc-mode` or `intero` to compile my Haskell).